modules/ding_content_rss/views/ding_content_rss_handler_library_id.inc

```diff
@@ -0,0 +1,45 @@
+<?php
+
+/**
+ * @file
+ * Node's "Library ID" custom field handler.
+ */
+
+/**
+ * Class ding_content_rss_handler_library_id.
+ */
+// @codingStandardsIgnoreStart
```

Comment:
I think that this "@codingStandardsIgnoreStart" and the "@codingStandardsIgnoreEnd" should be removed or the case why not should be given?

Reply:
Those are used here because PHPCS utility is recognizing non camel case case definition as a non-standard and failing Scrutinizer check. Those annotations are allowing us to skip any code standards checking on this piece of code.

Reply:
Yes, please remove them.

The guideline says that you should comment, in the PR, if scrutinizer throws errors that the core team should take into account when reviewing.

Please see https://github.com/ding2/ding2/blob/master/CONTRIBUTING.md point 10 and 11.

Reply:
Removed annotations.